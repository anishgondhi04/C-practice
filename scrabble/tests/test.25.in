10
2
2 1 V hello
3 1 V hello
2
2 l 8
2 o 5

