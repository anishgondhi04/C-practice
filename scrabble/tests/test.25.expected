Invalid word placement: (3,1) V, hello
..........
..h.......
..e.......
..l.......
..l.......
..o.......
..........
..........
..........
..........
Place horizontally at (2,5)
